packages/office-addin-manifest/test/test.ts

```diff
@@ -24,7 +24,7 @@ import { validateManifest } from "../src/validate";
 
 const manifestOriginalFolder = path.resolve("./test/manifests");
 const manifestTestFolder = path.resolve("./testExecution/testManifests");
-const testManifest = path.resolve(manifestTestFolder, "Taskpane.manifest.xml");
+const testManifest = path.resolve(manifestTestFolder, "TaskPane.manifest.xml");
```

Comment:
Why do we need this? Any change on the function name was made?

Reply:
`TaskPane.manifest.xml` is the real file name (See https://github.com/OfficeDev/Office-Addin-Scripts/blob/master/packages/office-addin-manifest/test/manifests/TaskPane.manifest.xml). Issue was introduced in 2018 and was the only thing which made the test on Linux fail since it will evaluate path in a case sensitive manner.